frontend/src/components/UI/TabPanels/Tabs.vue

```diff
@@ -11,6 +11,8 @@
 			>
 				<q-tab v-if="tags" dense name="tags" label="Tags" icon="sell" />
 				<q-tab dense name="products" label="Products" icon="apps" />
+				<q-tab dense name="childs" label="Childs" icon="mdi-source-branch" />
+				<q-tab dense name="gitconfig" label="Git config" icon="mdi-github" />
```

Comment:
This is not related to the pull request. Please, make another pull request for this.

Reply:
Ok